src/main/kotlin/com/goodchoice/rest/subject/SubjectController.kt

```diff
@@ -0,0 +1,53 @@
+package com.goodchoice.rest.subject
+
+import com.goodchoice.domain.common.model.Page
+import com.goodchoice.domain.common.model.PageRequest
+import com.goodchoice.domain.common.model.Reference
+import com.goodchoice.domain.subject.model.*
+import com.goodchoice.domain.subject.service.SubjectService
+import io.swagger.v3.oas.annotations.Operation
+import io.swagger.v3.oas.annotations.tags.Tag
+import org.springframework.http.HttpStatus
+import org.springframework.web.bind.annotation.*
+import java.util.*
+
+@RestController
+@RequestMapping("/subjects")
+@Tag(name = "subject")
+class SubjectController(private val subjectService: SubjectService) {
+
+    @GetMapping("/{id}")
+    @ResponseStatus(HttpStatus.OK)
+    @Operation(summary = "Get subject by id")
+    fun getById(@PathVariable(value = "id") id: UUID): Subject =
```

Comment:
Omit `(value = "id")` in @PathVariable if variable name equals path variable name